[13.0][IMP] ddmrp: Stock Buffers must be unique in product/location combination

BernatPForgeFlow opened the pull request.

BernatPForgeFlow committed:
[IMP] ddmrp: Stock Buffers must be unique in product/location combination

LoisRForgeFlow reviewed (approved):
LGTM

LoisRForgeFlow commented:
/ocabot merge patch

LoisRForgeFlow reviewed (commented):
Aborting merge, I just realized that the tests are failing and need fixing.

BernatPForgeFlow committed:
[IMP] ddmrp: Allow to duplicate multiple buffers

Select all the buffers to duplicate, determine the location and copy all the configuration.

LoisRForgeFlow reviewed (approved):
Aligned with v14

/ocabot merge minor
